app/helpers/govuk_design_system/summary_list_helper.rb

```diff
@@ -0,0 +1,47 @@
+module GovukDesignSystem
+  module GovukSummaryListHelper
+    # Use the [summary list](https://design-system.service.gov.uk/components/summary-list/)
+    # to summarise information, for example, a user’s responses at the end of a form.
+    #
+    # Implementation based on [nunjucks template](https://github.com/alphagov/govuk-frontend/blob/master/src/govuk/components/summary-list/template.njk)
+    def govukSummaryList(rows: [], classes: "", attributes: {})
```

Comment:
1. Can we add an alias to govuk_summary_list? seeing camel case in ruby code is a bit odd and the cognitive overload of swapping from camelcase to snake is fairly minimal. 
2. This has no test and I would be very reluctant to adopt this gem without a decent coverage.

Reply:
@StupidCodeFactory thanks for the comments.  My feeling was that it was more helpful to be consistent with the govuk-frontend nunjucks macros so code could be more easily copied and pasted from the prototyping kit or the examples on the Design System website.  A lowercase alias might be worth exploring, although some of the components also have snakedCased params, so it might be odd to mix those together.

I'd like to add tests, but need to spend some time figuring out the best way to do this. Ideally I think we'd reuse the tests from govuk-frontend.